src/libraries/Microsoft.PowerFx.Core/Types/FileType.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.Types
+{
+    public enum FileType
```

Comment:
>FileType [](http://example.com/codeflow?start=16&length=8)

We need very strong invariants about what these mean. 

For example, what's the penalty if we just always use Unknown?